monitor-openqa_job

```diff
@@ -31,7 +31,7 @@ declare -A failed_versions
 failed_jobs=()
 for job_id in $(job_ids job_post_response); do
     log-info "Waiting for job $job_id to finish"
-    $openqa_cli monitor --host "$host" --follow --poll-interval "$sleep_time" "$job_id"
+    $openqa_cli monitor --host "$host" --follow --poll-interval "$sleep_time" "$job_id" || true
```

Comment:
What is commonly done is the shorter

```suggestion
    $openqa_cli monitor --host "$host" --follow --poll-interval "$sleep_time" "$job_id" ||:
```

but I guess it would be nicer if the exit code is used to at least forward any problems encountered with a `log-info` or `log-warn`. How about that?

Reply:
the monitor command is only used to wait until the job is finished. The result of the job is retrieved with the next command.
If monitor fails, its error output will be in the log.
I could check the return code, and if it is 2, then the job has failed, if it is 1, then there was some other problem.
Should I do this?